src/show/pages/private/CreateProject/index.js

```diff
@@ -14,7 +14,7 @@ import {
   TeamMembersStep,
   TitleStep
 } from './Subviews';
-import { createProject } from '../../../../process/redux/projectSlice';
+import { createProject } from 'process/redux/projectSlice';
```

Comment:
The above import is relative and this one is absolute. They are group separately and reverse the order.